### PR TITLE
Add clap-sys-stubs pointer to README

### PR DESCRIPTION
**Pull request by spwilson2**

### CODE REVIEW

**Comment by spwilson2** [comment]
Definitely open to discussion on best way to share this! 

I personally got really annoyed trying copy-paste all the Option<extern "C"> fields to convert them into callback function definitions, so I wrote a short script to do it for me. I figure that essentially anyone using this crate will need to do the same (at least for creating a plugin), so it might be helpful to point them to a resource to do this automatically.


I'm also totally okay moving ownership to an org if there's multiple people who'd like to chip in to maintain.

---

Edit extending thoughts:

I just dug deeper into the [clap-plugins](https://github.com/free-audio/clap-plugins) reference and noticed its use of a [clap-helpers](https://github.com/free-audio/clap-helpers) repo. It might be more beneficial to actually provide a similar intermediary level of wrappers in addition to/instead of my stubs. 

Basically the wrappers in clap-helpers just add validation to make sure callbacks are executed on the main thread when they are supposed to etc, but with Rust that intermediate interface could also handle the translation of unsafe interfaces to safe ones.

**Comment by micahrj** [comment]
I appreciate you sharing your project!

> with Rust that intermediate interface could also handle the translation of unsafe interfaces to safe ones.

If you haven't seen it yet, this is actually the goal of the `clack` project: https://github.com/prokopyl/clack

I think the way that I'd like to do this is adding a "Related Projects" section to the README, with links to `clap-sys-stubs`, `clack`, and possibly some other projects (e.g. [`nih-plug`](https://github.com/robbert-vdh/nih-plug)). Again, I appreciate you sharing `clap-sys-stubs` and I bet people will find it useful!

**Comment by spwilson2** [comment]
Oh sweet! I hadn't heard about clack thanks for the pointer. (I basically only did a cursory google search for rust clap projects.)

Yeah it would be great for the rust + clap community to add a section in the readme for such pointers. (This page has the best SEO for such projects, so it would be a great place to do so.)
